pytket/pytket/passes/auto_rebase.py

```diff
@@ -19,7 +19,10 @@
 
 
 class NoAutoRebase(Exception):
-    """Automatic rebase could not be found."""
+    """Automatic rebase could not be found.
+
+    "DEPRECATED: will be removed after pytket 1.33."
```

Comment:
Other deprecations have suggested pytket 1.32 while this says pytket 1.33 - is this deliberate?

Reply:
Yeah just wanted to give a bit longer since some of the extensions still use `auto_rebase_pass`.